packages/angular_devkit/schematics/tools/export-ref_spec.ts

```diff
@@ -5,6 +5,7 @@
  * Use of this source code is governed by an MIT-style license that can be
  * found in the LICENSE file at https://angular.io/license
  */
+/// <reference types="@types/jasmine"/>
```

Comment:
instead of adding these, we should probably do the same as angular/angular where the tsconfig-test.json gives the jasmine types for free

/cc @gregmagolan

Reply:
removed reference and added tsconfig-test.json